README.md

```diff
@@ -33,12 +33,10 @@ To get more help on the Angular CLI use `ng help` or go check out the [Angular C
 
 ## Packaging 
 
-# for use in npm scripts
-npm install electron-packager --save-dev
+# Note regarding packaging
+Currently it is not possible to build Mac packages under other OS than Mac OS's. 
```

Comment:
The readme should include the commands that are used to produce the packages for each platform.

Reply:
ok